rosetta/identifier/operation.go

```diff
@@ -14,8 +14,8 @@
 
 package identifier
 
-// Operation uniquely identifies an operation within a transaction. We don't use
-// a network index because we don't have a sharded chain.
+// Operation uniquely identifies an operation within a transaction. No network index is
+// needed because of the absence of a shared chain.
```

Comment:
To be honest, the phrasing "we don't have a sharded chain" feels a bit clearer for me. Or "because Flow does not implement/support sharding" or something like that.

Reply:
```suggestion
// needed because of Flow does not support sharding.
```

👍